src/pieces.rs

```diff
@@ -308,7 +306,13 @@ impl Pieces {
         cr.translate(-0.5, -0.5);
         cr.scale(state.piece_set().scale(), state.piece_set().scale());
 
-        state.piece_set().by_piece(&figurine.piece).render_cairo(cr);
+        let renderer = librsvg::CairoRenderer::new(state.piece_set().by_piece(&figurine.piece));
+        renderer.render_element_to_viewport(cr, None, &Rectangle {
+            x: 0.0,
+            y: 0.0,
+            width: 177.0,
```

Comment:
I'm not sure about these either.
I haven't find the `render_cairo()` method.

Reply:
I think you'll want `.render_document(&cr, &Rectangle {...})` with librsvg's git master.  Sorry for the sudden change; this is to enable `.render_element()` as a new primitive.